### PR TITLE
Replace deprecated <script> attributes

### PR DESCRIPTION
**Pull request by jvoisin**

### CODE REVIEW

File: airsonic-main/cve-suppressed.xml [hunk 1]
```diff
@@ -213,4 +213,9 @@
         <gav regex="true">.*jackson-databind.*</gav>
         <cve>CVE-2019-14439</cve>
     </suppress>
+    <suppress>
+        <notes>We do not enable default typing for jackson</notes>
+        <gav regex="true">.*jackson-databind.*</gav>
+        <cve>CVE-2019-12384</cve>
```
Comment (eharris) [inline airsonic-main/cve-suppressed.xml]: It looks like this exclusion has already been applied to master.

File: airsonic-main/src/main/java/org/airsonic/player/spring/RegisterPrecompiledJSPInitializer.java [hunk 2]
```diff
@@ -64,6 +64,9 @@ private static WebApp parseXmlFragment() {
             jaxbContext = new JAXBDataBinding(WebApp.class).getContext();
             Unmarshaller unmarshaller = jaxbContext.createUnmarshaller();
             WebApp webapp = (WebApp) unmarshaller.unmarshal(webXmlIS);
+            try {
+                webXmlIS.close();
+            } catch (java.io.IOException e) {}
```
Comment (eharris) [inline airsonic-main/src/main/java/org/airsonic/player/spring/RegisterPrecompiledJSPInitializer.java]: I would prefer this use `ignored`

File: airsonic-main/src/main/webapp/WEB-INF/jsp/playQueue.jsp [hunk 3]
```diff
@@ -550,11 +550,11 @@
...
-                if (song.starred) {
-                    $("#starSong" + id).attr("src", "<spring:theme code='ratingOnImage'/>");
-                } else {
-                    $("#starSong" + id).attr("src", "<spring:theme code='ratingOffImage'/>");
-                }
+            if (song.starred) {
+                $("#starSong" + id).attr("src", "<spring:theme code='ratingOnImage'/>");
+            } else {
+                $("#starSong" + id).attr("src", "<spring:theme code='ratingOffImage'/>");
+            }
```
Comment (eharris) [inline airsonic-main/src/main/webapp/WEB-INF/jsp/playQueue.jsp]: Wrong indent

File: airsonic-main/src/main/webapp/WEB-INF/jsp/playQueue.jsp [hunk 4]
```diff
@@ -600,7 +600,7 @@
                 // If song.year is not an int, this will return NaN, which
                 // conveniently returns false in all boolean operations.
                 if (parseInt(song.year) > 0) {
-                    $("#year" + id).text(song.year);
+                $("#year" + id).text(song.year);
```
Comment (eharris) [inline airsonic-main/src/main/webapp/WEB-INF/jsp/playQueue.jsp]: Wrong indent

**Comment by fxthomas** [comment]
I don't see crashes or anything when running this. Should be good to merge!

File: airsonic-main/src/main/webapp/WEB-INF/jsp/head.jsp [hunk 5]
```diff
@@ -19,7 +19,7 @@
 <script id="preferencesConfig" type="application/x-configuration">
   {
     "keyboardShortcutsEnabled": ${model.keyboardShortcutsEnabled ? 'true' : 'false'}
-  }
+}
```
Comment (eharris) [inline airsonic-main/src/main/webapp/WEB-INF/jsp/head.jsp]: Indent

**Comment by jvoisin** [comment]
This one should be merged _after_ #1350

**Comment by fxthomas** [comment]
We can probably rebase and merge it now ;)

**Comment by eharris** [comment]
Why was this closed?

**Comment by jvoisin** [comment]
Because it would be easier to run `sed` again than to resolve all the conflicts.

**Pull request closed without merging**